components/custom_status/custom_status_suggestion.tsx

```diff
@@ -37,6 +37,7 @@ const CustomStatusSuggestion: React.FC<Props> = (props: Props) => {
             handleClear(status);
         }
     };
+    const customStatusDuration = CustomStatusDuration;
```

Comment:
this is not needed ... just check against the `CustomStatusDuration` enum properties